src/main/resources/swagger.api/schemas/dto/staffSlips/staffSlipsCollection.yaml

```diff
@@ -0,0 +1,13 @@
+description: "Collection of staff slips"
+type: "object"
+properties:
+  totalRecords:
+    type: "integer"
```

Comment:
You're using quotes here for types but not in the staffSlip.yaml

Reply:
removed quotes